File: .sdk/tm/csharp/utility/PrepareBody.cs
// Hook0 SDK utility: prepareBody.

namespace Hook0Sdk.Util;

public static partial class SdkUtility
{
    internal static object? PrepareBodyUtil(Context ctx)
    {
        var op = ctx.Op!;

        if (op.Input == "data")
        {
            var body = ctx.Utility!.TransformRequest(ctx);
            return body;
        }

        return null;
    }
}

File: php/utility/PrepareBody.php
<?php
declare(strict_types=1);

// Hook0 SDK utility: prepare_body

class Hook0PrepareBody
{
    public static function call(Hook0Context $ctx): mixed
    {
        if ($ctx->op->input === 'data') {
            return ($ctx->utility->transform_request)($ctx);
        }
        return null;
    }
}
